templates/CRM/Financeextras/Form/Payment/Refund.tpl

```diff
@@ -0,0 +1,66 @@
+<div class="crm-block crm-form-block crm-payment-refund-form-block">
+<div class="crm-accordion-body">
+<div class="message help" >{ts}Refunds take 5-10 days to appear on a customer's statement{/ts}</div></div>
+ <table class="form-layout-compressed">
+        <tr class="crm-payment-refund-form-block-contact_id">
+          <td class="label">{$form.contact.label}</td>
+          <td>{$form.contact.html}</td>
+        </tr>
+        <tr class="crm-payment-refund-form-block-contact_id">
+          <td class="label"><label>{ts}Select Payment To Refund{/ts}</label></td>
+          <td><table class="selector row-highlight payment-info">
+          <tr>
+          <th></th>
+            <th>{ts}Date{/ts}</th>
+            <th>{ts}Original Amount{/ts}</th>
+            <th>{ts}Available Amount{/ts}</th>
+            <th>{ts}Payment Processor{/ts}</th>
+            <th>{ts}Processor Payment ID{/ts}</th>
+          </tr>
+        {foreach from=$paymentInfos item=paymentInfo}
+              <tr  id="tr_{$paymentInfo.financialTrxnId}">
+                <td><input class=" required crm-form-radio" value="{$paymentInfo.transactionId}" type="radio" id="{$paymentInfo.financialTrxnId}" name="payment_row" ></td>
+                <td>{$paymentInfo.date}</td>
+                <td>{$paymentInfo.amount|crmMoney:$paymentInfo.currency}</td>
+                <td class="available_amount_{$paymentInfo.financialTrxnId}">{$paymentInfo.available_amount|crmMoney:$paymentInfo.currency}</td>
+                <td>{$paymentInfo.paymentProcessor}</td>
+                <td>{$paymentInfo.transactionId}</td>
+              </tr>
+          {/foreach}
+</table>
+</td>
+        </tr>
+        <tr  class="crm-payment-refund-form-block-total_amount">
+            <td class="label">{$form.amount.label}</td>
+            <td {$valueStyle}>
+              <span id='totalAmount'>{$form.currency.html|crmAddClass:eight}&nbsp;{$form.amount.html|crmAddClass:eight}</span>
+              {if $freezeFinancialType}
+                {help id="id-amount"}
+              {/if}
+              {if !$payNow}
+                {if $hasPriceSets}
+                  <span id='totalAmountORPriceSet'> {ts}OR{/ts}</span>
+                  <span id='selectPriceSet'>{$form.price_set_id.html}</span>
+                  <div id="priceset" class="hiddenElement"></div>
+                {/if}
+
+                {if $ppID}{ts}<a class='action-item crm-hover-button' onclick='adjustPayment();'>adjust payment amount</a>{/ts}{help id="adjust-payment-amount"}{/if}
+                <div id="totalAmountBlock">
+                  {if $hasPriceSets}<span class="description">{ts}Alternatively, you can use a price set.{/ts}</span>{/if}
```

Comment:
How priceset is being used  here?

Reply:
@erawat We have removed priceset and display only currency.